doc/tutorials/dnn/dnn_android/dnn_android.markdown

```diff
@@ -8,15 +8,15 @@
 |    |    |
 | -: | :- |
 | Original author | Dmitry Kurtaev |
-| Compatibility | OpenCV >= 3.3 |
+| Compatibility | OpenCV >= 4.8 |
```

Comment:
I recommend to use 4.9.0. It's the first release version.

Reply:
fixed